lib/game.rb

```diff
@@ -0,0 +1,50 @@
+require '../lib/interface.rb'
+
+class Game
+  include Interface
+  attr_accessor :symbol
+
+  def initialize(board, player_1, player_2)
+    @board = board
+    @player_1 = player_1
+    @player_2 = player_2
+    @turn = 1
+   end
+
+  def win?(item)
+    temp = []
+    @board.board.each_with_index do |value, index|
+      temp << index if value == item
+    end
+    @board.win_patterns.each do |arr|
+      result = arr - temp
+      return true if result.empty?
+    end
+    false
+  end
+
+
+	def gameplay
```

Comment:
It appears `gameplay` is the only method of `Game` that is being used by other objects (`executable.rb`). So why don't we place all other methods under `private`?